app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -231,9 +245,15 @@ export default function ManageChannelMembers({
 
     useNavButtonPressed(MANAGE_BUTTON, componentId, toggleManageEnabled, [toggleManageEnabled]);
 
-    useEffect(() => {
-        mounted.current = true;
-        const options: GetUsersOptions = {sort: 'admin', active: true, per_page: PER_PAGE_DEFAULT};
+    const handleReachedBottom = useCallback(() => {
+        if (hasMoreProfiles.current && !loading && !searchedTerm) {
+            setLoading(true);
+            setPage((p) => p + 1);
```

Comment:
You can call here the getFetchChannelMembers instead of waiting for the next render.